Sources/Testing/Testing.docc/DefiningTests.md

```diff
@@ -87,6 +87,6 @@ a test is unable to run due to limited availability:
 
 ```swift
 @available(macOS 11.0, *)
-@available(swift, introduced: 8.0, message: "Requires Swift 8.0 features to run")
+@available(swift, introduced: 5.0, message: "Requires Swift 5.0 features to run")
```

Comment:
Please change this back. It intentionally lists a (hypothetical) future version of Swift. We do not support Swift 5.0.

Reply:
Will revert.